pilot/pkg/proxy/envoy/v1/testdata/lds-mixer-check-report-config.json.golden

```diff
@@ -0,0 +1,891 @@
+{
+  "listeners": [
+   {
+    "address": "tcp://0.0.0.0:15001",
+    "name": "virtual",
+    "filters": [],
+    "bind_to_port": true,
+    "use_original_dst": true
+   },
+   {
+    "address": "tcp://0.0.0.0:443",
+    "name": "http_0.0.0.0_443",
+    "filters": [
+     {
+      "type": "read",
+      "name": "http_connection_manager",
+      "config": {
+       "codec_type": "auto",
+       "stat_prefix": "http",
+       "generate_request_id": true,
+       "tracing": {
+        "operation_name": "egress"
+       },
+       "rds": {
+        "cluster": "rds",
+        "route_config_name": "443",
+        "refresh_delay_ms": 10
+       },
+       "filters": [
+        {
+         "type": "decoder",
+         "name": "mixer",
+         "config": {
+          "mixer_attributes": {
+           "destination.ip": "10.1.1.0",
+           "destination.service": "hello.default.svc.cluster.local",
+           "destination.uid": "kubernetes://v0.default"
+          },
+          "forward_attributes": {
+           "source.ip": "10.1.1.0",
+           "source.uid": "kubernetes://v0.default"
+          },
+          "quota_name": "RequestCount",
+          "v2": {
+           "defaultDestinationService": "hello.default.svc.cluster.local",
+           "forwardAttributes": {
+            "attributes": {
+             "source.ip": {
+              "bytesValue": "AAAAAAAAAAAAAP//CgEBAA=="
+             },
+             "source.uid": {
+              "stringValue": "kubernetes://v0.default"
+             }
+            }
+           },
+           "mixerAttributes": {
+            "attributes": {
+             "destination.ip": {
+              "bytesValue": "AAAAAAAAAAAAAP//CgEBAA=="
+             },
+             "destination.uid": {
+              "stringValue": "kubernetes://v0.default"
+             }
+            }
+           },
+           "serviceConfigs": {
+            "hello.default.svc.cluster.local": {
+             "disableCheckCalls": true,
+             "disableReportCalls": true,
```

Comment:
@ayj is this the default? this looks very suspicious..

Reply:
never mind. found the issue and fixed.

Reply:
actually no.. the issue persists..

Reply:
Ignore again.. This seems to be set throughout the code.

Reply:
Check/Report should be disabled for outbound requests. Unfortunately, the polarity of this configuration is super confusing since it defaults to enabled when the value is "false" and we end up needing to explicitly disabled it by setting values to `true` in a bunch of places. We should consider inverting the logic and add new options for `enableXXXCalls`, update the proxy,/mixerclient to use them instead, and deprecate/remove the `disableXXXCalls`. cc @qiwzhang